ed448-goldilocks/src/field/element.rs

```diff
@@ -463,16 +463,16 @@ mod tests {
                 (84 * 2).try_into().unwrap(),
             )
             .unwrap();
-            let mut data = Array::<u8, U84>::default();
-            expander.fill_bytes(&mut data);
+            let mut data = Array::<u8, U84>::from_iter(expander.by_ref().take(84));
             // TODO: This should be `Curve448FieldElement`.
             let u0 = Ed448FieldElement::from_okm(&data).0;
```

Comment:
```suggestion
            // TODO: This should be `Curve448FieldElement`.
            let u0 = Ed448FieldElement::from_okm(&expander.by_ref().take(84).collect()).0;
```
We can do this in a lot of other places as well.